[14.0] [IMP] l10n_it_riba_sale_commission: use wizard's payment date (#4284)
OCA/l10n-italy: l10n_it_riba_sale_commission/README.rst

It is possible to set "Without commissions" flag in invoices, in this way
no agent commissions are generated.

When generating agent settlements, select a payment date until which commissions should be settled.

> Invoices with RiBa payment will be taken into consideration only if the number of days set in the "Safety days for commission" field in RiBa configuration from due date set in RiBa payment have passed from settlement payment date.

Reviewer: I find this sentence a bit hard to read. What do you think about this proposal?

```suggestion
Invoices with RiBa payment will be taken into consideration only if enough days (set in the "Safety days for commission" field in RiBa configuration) have passed since the  "Due date" of a settlement payment.
```

Reviewer 2: Invoices with RiBa payment will be taken into consideration only if enough days (set in the "Safety days for commission" field in RiBa configuration) have passed between the "due date" of the RiBa payment and the payment date set in the settlement wizard.
